docs/testing.rst

```diff
@@ -73,20 +73,18 @@ have bedrock running at ``localhost`` on port ``8000``.
 To run a single test suite you must tell CasperJS to execute a specific file
 e.g. ``tests/functional/home.js``::
 
-    PHANTOMJS_EXECUTABLE=./node_modules/.bin/phantomjs ./node_modules/.bin/casperjs test tests/functional/home.js --config=tests/config.json
+    npm test -- tests/functional/home.js --config=tests/config.json
 
 You can also easily run the tests against any bedrock environment by specifying the
 ``--domain`` variable. For example, to run all functional tests against dev::
 
-    PHANTOMJS_EXECUTABLE=./node_modules/.bin/phantomjs ./node_modules/.bin/casperjs test tests/functional --domain=https://www-dev.allizom.org --config=tests/config.json
+    npm test -- tests/functional --domain=https://www-dev.allizom.org --config=tests/config.json
 
 .. Note::
 
-    ``npm test`` is just a simple shortcut and not something you can pass options to. If
-    you want to specify a particular file to run or a different domain, you must use the
-    full command like in the examples above. Also, make sure to include the configuration
-    file ``--config=tests/config.json`` when running tests, as this is sometimes needed
-    for PhantomJS to open certain URLs served over https.
+    Passing arguments to ``npm test`` requires at least npm v2.0.0. If you're using an
+    older version of npm, you can still run the tests manually, instead of
+    using the ``npm test`` shortcut.
```

Comment:
I don't see reason to repeat it again when it's shown just above, but I've updated the copy to be a little clearer hopefully. Let me know what you think